packages/core/src/tracing/spanHandle.js

```diff
@@ -304,6 +419,8 @@ NoopSpanHandle.prototype.setCorrelationType = function setCorrelationType() {};
 
 NoopSpanHandle.prototype.annotate = function annotate() {};
 
+NoopSpanHandle.prototype.markAsError = function markAsError() {};
```

Comment:
I guess this must be renamed too?

Reply:
Oh, very good catch. Fixed and I added a test to catch missing methods on NoopSpanHandle automatically in the future, so this should not happen again.